tests/components/switch/test_flux.py

```diff
@@ -183,22 +183,24 @@ def event_date(hass, event, now=None):
                 return sunrise_time
             return sunset_time
 
-        with patch('homeassistant.util.dt.now', return_value=test_time):
-            with patch('homeassistant.helpers.sun.get_astral_event_date',
-                       side_effect=event_date):
-                assert setup_component(self.hass, switch.DOMAIN, {
-                    switch.DOMAIN: {
-                        'platform': 'flux',
-                        'name': 'flux',
-                        'lights': [dev1.entity_id]
-                    }
-                })
-                turn_on_calls = mock_service(
-                    self.hass, light.DOMAIN, SERVICE_TURN_ON)
-                common.turn_on(self.hass, 'switch.flux')
-                self.hass.block_till_done()
-                fire_time_changed(self.hass, test_time)
-                self.hass.block_till_done()
+        with patch('homeassistant.util.dt.now', return_value=test_time), \
```

Comment:
We should be able to remove this patch now since we patch in the correct place below, right?

Reply:
Good point 👍